release-notes/8.0/supported-os.json

```diff
@@ -0,0 +1,20 @@
+{
+  "channel-version": "8.0",
+  "last-updated": "2024-02-28",
+  "operating-systems" : [
+    {
+      "name": "Ubuntu",
+      "architectures": ["x64", "arm64", "arm"],
+      "supported-versions" : ["20.04", "22.04", "23.10", "24.04"],
+      "unsupported-versions" : ["18.04"],
```

Comment:
.NET 8 never supported `16.04`, `22.10` or `23.04` (which was once supported by .NET 7 and is EOS now).